aubrey_transcription/default_settings.py

```diff
@@ -9,3 +9,4 @@
 }
 FILENAME_PATTERN = (r'(?P<metaid>[^_]*)_m?(?P<manifestation>[^_]*)_(?P<fileset>[^-]*)'
                     r'-(?P<kind>[^-]*)-(?P<language>[^.]*)\.(?P<extension>.*)')
+JSON_SORT_KEYS = False
```

Comment:
I don't believe so, since this isn't something that should be adjusted and will be loaded in from these default settings.

Reply:
If someone goes by the alternative option in the README where it says to "[create] a directory called "instance" at the root of the project and [put] a new file called "settings.py" there with the following 5 settings", wouldn't they also need to include `JSON_SORT_KEYS`?

Reply:
nope, because I load settings with an order of priority, so the setting should still be there even if they create a that file, because I first load the defaults and then start overriding those loaded settings with others if present. [source](https://github.com/unt-libraries/aubrey-transcription/blob/master/aubrey_transcription/__init__.py#L18)

Reply:
Ok then! :)